File: question8.java
import java.util.Scanner;
public class question8 {
    public static void main(String[] args) {
        //bolme islemini cikarma kullanarak yapan kod

        Scanner scanner = new Scanner(System.in);

        System.out.print("Bolunen sayiyi giriniz:");
        int bolunen = scanner.nextInt();

        System.out.print("Bolen sayiyi giriniz:");
        int bolen = scanner.nextInt();

        int sonuc=0;

        while(bolunen>=bolen){
            bolunen-=bolen;
            sonuc++;
        }

        System.out.print("Bolme isleminin sonucu:" + sonuc);

        scanner.close();
    }
}

File: question8.py
# bolme islemini cikarma kullanarak bulan kod

bolunen=int(input("Bolunen sayiyi giriniz:"))
bolen=int(input("Bolen sayiyi giriniz:"))
sonuc=0

while(bolunen>=bolen):
    bolunen=bolunen-bolen
    sonuc=sonuc+1

print("Bolme isleminin sonucu:" + str(sonuc))